packages/react-server-module-tagger/index.js

```diff
@@ -0,0 +1,129 @@
+var isWindows = ('win32' === process.platform);
+
+/**
+ * A util function for tagging modules.  Expects to find data on its prototype
+ * as follows
+ *     {
+ *         file: {
+ *             path: the path to the file to tag
+ *         }
+ *         config: {
+ *             trim: the string to trim off the front of the logger name
+ *         }
+ *     }
+ * @example
+ *     const file = '/path/to/my/file.js';
+ *     loggerSpec.bind({file, { trim: 'path.to.' }})(file, '({label: "foo"})')
+ *     // returns '{\"label\":\"foo\",\"name\":\"my.file\",\"color\":{\"server\":87,\"client\":\"rgb(42,212,212)\"}}'
+ * @param  {String} fullMatch May also be provided as this.file.path, the path to the file
+ * @param  {String} optString The label to add to the module tag, in the form '({label:"$label"})'
+ * @return {String}           A json object containing a module identifier
+ */
+module.exports = function(fullMatch, optString){
+	var fn   = this.file && this.file.path ? this.file.path : fullMatch
```

Comment:
`file` isn't used for anything other than the `path` in here, right?  Might be nicer to just take the path rather than the file object, which is particular to gulp.